uint16 traj_id
---
uint8 success